<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>条码打印</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/images.qrc">
    <normaloff>:/logo/logo64.ico</normaloff>:/logo/logo64.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>377</y>
      <width>571</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>201</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>操作区</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>173</width>
       <height>225</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_select_capture_region_2">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_select_capture_region_3">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_select_capture_region_4">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>手动读号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_select_capture_region">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>选择识别区域</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_show_capture">
        <property name="minimumSize">
         <size>
          <width>99</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>查看识别区域</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>41</y>
      <width>413</width>
      <height>319</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>项目:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_product">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>钥匙号:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>411</width>
         <height>191</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>411</width>
         <height>191</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>90</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>173</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PLC的IP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_IP_PLC">
       <property name="text">
        <string>192.168.0.255</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_change_ip">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1.2, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(200, 194, 15);</string>
       </property>
       <property name="text">
        <string>设置为当前值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_plc_connect_test">
       <property name="minimumSize">
        <size>
         <width>99</width>
         <height>28</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文楷体</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1.2, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(200, 194, 15);</string>
       </property>
       <property name="text">
        <string>测试连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_change_ip</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>change_ip_plc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_plc_connect_test</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>test_connect_plc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_select_capture_region</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_capture_region()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_show_capture</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_capture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_select_capture_region_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_select_capture_region_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_select_capture_region_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>manual_key_read()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>change_ip_plc()</slot>
  <slot>test_connect_plc()</slot>
  <slot>select_capture_region()</slot>
  <slot>show_capture()</slot>
  <slot>start()</slot>
  <slot>pause()</slot>
  <slot>manual_key_read()</slot>
 </slots>
</ui>
